<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1470</width>
    <height>536</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ADC Control</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2" rowspan="4">
    <widget class="QwtPlot" name="graphD">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>480</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QwtPlot" name="graphA">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>480</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QwtPlot" name="graphC">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>240</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QwtPlot" name="graphB">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>240</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="editStd">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
